src/modules/auth.js

```diff
@@ -25,9 +25,7 @@ const performAuthentication = async (e, dispatch) => {
       });
       localStorage.removeItem("J-tockAuth-Storage");
     }
-  } 
-  catch (error) {
-    console.log(error);
+  } catch (error) {
```

Comment:
Here you removed the cosole.log(error), is there any particulare reason for that?

Reply:
It's because we have a dispatch function that set error message when error is caught, so we thought console.log is not needed anymore.